quolance-api/src/main/java/com/quolance/quolance_api/controllers/ClientController.java

```diff
@@ -55,6 +56,19 @@ public ResponseEntity<String> deleteProject(@PathVariable(name = "projectId") Lo
         return ResponseEntity.ok("Project deleted successfully");
     }
 
+    @PutMapping("/projects/{projectId}")
+    @Operation(
+            summary = "Update a project",
+            description = "Update an existing project by passing the project details"
+    )
+    public ResponseEntity<ProjectDto> updateProject(
+            @PathVariable(name = "projectId") Long projectId,
+            @RequestBody ProjectUpdateDto projectUpdateDto) {
+        User client = SecurityUtil.getAuthenticatedUser();
+        ProjectDto updatedProject = clientWorkflowService.updateProject(projectId, projectUpdateDto, client);
```

Comment:
remove projectId add to dto

Reply:
Jcomprends pas pq

Reply:
C'est mieux de separer resource identification et modification